config/src/config/mempool_config.rs

```diff
@@ -22,17 +22,13 @@ pub struct LoadBalancingThresholdConfig {
     /// greater than `avg_mempool_traffic_threshold_in_tps`, then the PFN will forward mempool traffic to only those upstream peers
     /// with ping latency less than `x + latency_slack_between_top_upstream_peers`.
     pub latency_slack_between_top_upstream_peers: u64,
-    /// If the average received mempool traffic is greater than avg_mempool_traffic_threshold_in_tps, then PFNs will forward to at most
-    /// `max_number_of_upstream_peers` upstream FNs.
-    pub max_number_of_upstream_peers: u8,
```

Comment:
If this config is removed, we need to modify the corresponding code in priority.rs as well to remove the usage of it.

Reply:
Removing this limit means the PFNs could be distributing load to 4 VFNs even at low latencies. This means more signed batches are created and need to be verified, etc. Is that fine?